Automattic/wp-calypso
# Customer Home: add editor deprecation notice

#### Changes proposed in this Pull Request

* Adds a notice about the upcoming editor deprecation to the site home for users still on classic editor

#### Testing instructions
* Checkout this PR 
* Go to Home of site which has classic editor set and with the config flag `home/layout-dev` set to true (enabled on development by default). 
* Check that the "Block editor is coming" message displays
* Check that the "Try it now" button takes you to gutenberg and sets it as editor (so returning to home should no longer display the notice"
* Check that the Dismiss link works
* Check that this does not appear when `home/layout-dev` is set to false. This can also be set as a query param `?flags=-home/layout-dev`

Home with classic editor set:

<img width="936" alt="Screen Shot 2020-05-05 at 11 47 03 AM" src="https://user-images.githubusercontent.com/3629020/81024019-3578cf00-8ec6-11ea-93b9-b93cb31f79f5.png">

Relates to  #41085
